eo-maven-plugin/src/main/java/org/eolang/maven/hash/CommitHash.java

```diff
@@ -23,12 +23,15 @@
  */
 package org.eolang.maven.hash;
 
+import org.cactoos.Scalar;
+
 /**
  * Hash of tag.
  *
  * @since 0.28.11
  */
-public interface CommitHash {
+@FunctionalInterface
```

Comment:
@levBagryansky why do we need it here?

Reply:
@maxonfjvipon I added it because this actually is a functional interface.